lib/code_corps_web/controllers/project_github_repo_controller.ex

```diff
@@ -1,24 +1,31 @@
 defmodule CodeCorpsWeb.ProjectGithubRepoController do
   use CodeCorpsWeb, :controller
-  use JaResource
 
-  import CodeCorps.Helpers.Query, only: [id_filter: 2]
+  alias CodeCorps.{ProjectGithubRepo, User, Helpers.Query}
 
-  alias CodeCorps.ProjectGithubRepo
+  action_fallback CodeCorpsWeb.FallbackController
+  plug CodeCorpsWeb.Plug.DataToAttributes
+  plug CodeCorpsWeb.Plug.IdsToIntegers
 
-  plug :load_resource, model: ProjectGithubRepo, only: [:show], preload: [:github_repo, :project]
-  plug :load_and_authorize_changeset, model: ProjectGithubRepo, only: [:create]
-  plug :load_and_authorize_resource, model: ProjectGithubRepo, only: [:delete]
```

Comment:
@landongrindheim As you can see, we do actually need to support a delete action. 

It's not clear here, but we likely also need to support an index action (it's implicit from us having a `filter` function before). These are the reasons we're switching away from `ja_resource`. Lot's of hidden stuff.

Basically, the controller needs to support

- index (with id filter)
- show
- create (with authorization)
- delete (with authorization)

Reply:
You can use `OrganizationGithubAppInstallationController` as a sort of template.